### PR TITLE
Added new functions to download_dataport.py

### LINKED ISSUE #724: How to convert 1Hz data of dataport into H5 format？
By default, nilmtk downloads data ports for 1 minute. Now that dataport already have 1hz data, I want to know how to convert 1Hz data into H5 format. Thank you. Will it be a problem to directly replace electricity_egauge_minutes in the source code with electricity_egauge_seconds?

### PR DESCRIPTION
**Pull request by raktim2015**
Closes #724 and #742

### CODE REVIEW

File: nilmtk/dataset_converters/dataport/download_dataport.py [hunk 1]
```diff
@@ -128,6 +128,37 @@ def database_assert(database_table):
...
+        print('Could not connect to remote database')
+        raise
+
+    
+    #Loading university schemas
+    sql_query = ("SELECT table_name" +
+                 " FROM information_schema.views" +
+                 " WHERE table_schema ='" + database_schema + "'" +
+                 " ORDER BY table_name")
+    database_tables=pd.read_sql(sql_query, conn)['table_name'].tolist()
```
Comment (nipunbatra) [inline nilmtk/dataset_converters/dataport/download_dataport.py]: ensure pep8 please

File: nilmtk/dataset_converters/dataport/download_dataport.py [hunk 2]
```diff
@@ -128,6 +128,37 @@ def database_assert(database_table):
...
+
+    df=pd.DataFrame({database_schema:database_tables})
+    #print(database_tables)
```
Comment (nipunbatra) [inline nilmtk/dataset_converters/dataport/download_dataport.py]: remove all such comments

**Comment by nipunbatra** [comment]
Excellent work. I'm merging this PR. Requesting to pull the changes after I accept this PR and open another PR for making PEP8 changes in the files you changed.

File: nilmtk/dataset_converters/dataport/download_dataport.py (hunk 1 above)
Comment (raktim2015) [reply nilmtk/dataset_converters/dataport/download_dataport.py]: Sure sir. Should I ensure pep8 after you merge the pull request?

File: nilmtk/dataset_converters/dataport/download_dataport.py (hunk 2 above)
Comment (raktim2015) [reply nilmtk/dataset_converters/dataport/download_dataport.py]: Okay. I am adding another commit soon

Comment (raktim2015) [reply nilmtk/dataset_converters/dataport/download_dataport.py]: I will be doing both of it in the new PR

**Pull request merged**